### PR TITLE
feat: add rich UI on boost

### PR DESCRIPTION
**Pull request by ayushtom**
Screenshot 2023-12-14 at 4 20 38 PM
Made rich UI on boost cards on expiry

### CODE REVIEW

File: components/quest-boost/boostCard.tsx [hunk 1]
```diff
@@ -35,10 +35,10 @@ const BoostCard: FunctionComponent<BoostCardProps> = ({ boost }) => {
           </div>
           <div className="flex w-full">
             {boost.expiry > Date.now() ? null : (
-              <div className="flex items-center opacity-40">
+              <div className="flex items-center">
                 <div className={styles.issuer}>
-                  <p className="text-white mr-2">Expired</p>
-                  <UnavailableIcon width="24" color="#D32F2F" />
+                  <p className="text-white mr-2">Boost Ended</p>
+                  <CheckIcon width="24" color="#6AFFAF" />
```
Comment (fricoben) [inline components/quest-boost/boostCard.tsx]: Imo we should still have something red showing it's not feasible anymore

**Review by fricoben — CHANGES_REQUESTED** [review]
Small change and we're good !

File: components/quest-boost/boostCard.tsx (hunk 1 above)
Comment (ayushtom) [reply components/quest-boost/boostCard.tsx]: done

**Comment by ayushtom** [comment]
Screenshot 2023-12-14 at 11 03 24 PM

Updated View

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**